CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [1.7.0](https://github.com/michael-mosler/Pius-App-for-Android/compare/v1.6.0...1.7.0) - 2021-04-10
+### Added
+- Autofill support on login
+- The Staff-Popover now contains the email-address.
+### Changed
+- UI-Design of the app
+	- The buttons now have a rounded design.
+	- The settings are now split in tabs.
+	- The In-App Changelog now appears in a Pop-Up (instead of full-screen)
+	- The size of the ticker text window is now adapted to the size of the ticker text.
+### Fixed
+- Fixed a bug that caused entries in the course list to show off duplicated.
```

Comment:
Shouldn't we also mention fix of bug #36 ?

Reply:
The bug was only caused by a change and never existed in version 1.6.0. I therefore do not consider it necessary to mention it.